### DISCUSSION TITLE
吉林大学 - Scholar Compass 高校文档

### DISCUSSION THREAD

**Discussion by giscus** [Scholar-Compass/content#11] [General]
# 吉林大学 - Scholar Compass 高校文档

帮助高三毕业生更好地择校选专业。

https://scholar-compass.github.io/content/university/%E5%90%89%E6%9E%97%E5%A4%A7%E5%AD%A6/?h=%E5%90%89%E6%9E%97

**Comment by YDX-2147483647** [comment]
有个链接无法访问。

https://github.com/Scholar-Compass/content/blob/f9e3452a378c216b150bd20bdd46699baf6ccc00/docs/university/%E5%90%89%E6%9E%97%E5%A4%A7%E5%AD%A6.md?plain=1#L76-L77

根据[校内vpn导航-大数据和网络管理中心 (nic.jlu.edu.cn)](https://nic.jlu.edu.cn/info/1016/2037.htm)，`oa.jlu.edu.cn`恐怕仅限校内访问。

https://github.com/Scholar-Compass/content/issues/10
